application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -219,44 +210,45 @@ List<TripTimeOnDate> stopTimesForPatternAtStop(
     boolean includeCancellations
   );
 
-  Collection<GroupOfRoutes> getGroupsOfRoutes();
+  Collection<GroupOfRoutes> listGroupsOfRoutes();
 
-  Collection<Route> getRoutesForGroupOfRoutes(GroupOfRoutes groupOfRoutes);
+  Collection<Route> findGroupsOfRoutes(GroupOfRoutes groupOfRoutes);
```

Comment:
I think the name should be `findRoutes` but I prefer a longer version like `findRoutesForGroupOfRoutes`.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6249/commits/5bf1fc09a35a82e744f2b18e247bc68e5687af08

Reply:
I prefer the short version, because the argument is part of the method signature - [DRY](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/DEVELOPMENT_DECISION_RECORDS.md#dry---do-not-repeat-yourself)